### PR TITLE
Open changelog url when `o` key is pressed

### LINKED ISSUE #30: Feature Idea: Ability to open changeling URL with a keypress
While I'm looking through the list of changes it would be convenient to be able to press `o` or some other key to open the changelog URL of the currently selected gem.

### PR DESCRIPTION
**Pull request by mattbrictson**
Add a key binding for opening the changelog URL. Now you can navigate to a gem with up/down and press `o` (lower-case letter o) to open the URL in your default web browser.

This implementation depends on the [launchy](https://github.com/copiousfreetime/launchy) gem. Refer to its README for information on configuring which browser is used.

TODO

- [x] PR description
- [x] Unit test
- [x] Update README image
- [x] Fix failing Ruby head test due to undeclared launchy → childprocess → logger dependency

Closes #30

### CODE REVIEW

**Comment by mattbrictson** [comment]
@spilth does this look good to you?

**Comment by dylnclrk** [comment]
What a neat feature!

File: images/update-interactive.png
Comment (dylnclrk) [inline images/update-interactive.png:1]: Do you have a procedure for creating these?

Comment (mattbrictson) [reply images/update-interactive.png:1]: @dylnclrk my procedure is very low-tech: I take a new screenshot and then cut and paste the new feature on top of the old one. So in this case a pasted this slice on top of the old image:

Screenshot 2024-08-03 at 5 01 31 PM

That way I don't have to recreate the circumstances of the original screenshot. 🙂

**Comment by spilth** [comment]
> @spilth does this look good to you?

Very cool - thank you!

**Pull request merged**